Tasks/Common/VstsAzureHelpers_/ImportFunctions.ps1

```diff
@@ -76,17 +100,15 @@ function Import-FromModulePath {
             $script:azureModule = $module
         } else {
             # The AzureRM module was imported.
-
             # Validate the AzureRM.profile module can be found.
-            $profileModule = Get-Module -Name AzureRM.profile -ListAvailable | Select-Object -First 1
+            $profileModule = (Get-Module -Name AzureRM).NestedModules | Where-Object { $_.Name.toLower() -eq "azurerm.profile" }
```

Comment:
will suggest to put this in try catch from line 104-111 will help to debug in future